crates/miden-lib/asm/kernels/transaction/lib/epilogue.masm

```diff
@@ -332,5 +344,5 @@ export.finalize_transaction
     # => [OUTPUT_NOTES_COMMITMENT, FINAL_ACCOUNT_COMMITMENT]
 
     exec.memory::get_expiration_block_num movdn.8
-    # => [OUTPUT_NOTES_COMMITMENT, FINAL_ACCOUNT_HASH, tx_expiration_block_num]
+    # => [OUTPUT_NOTES_COMMITMENT, FINAL_ACCOUNT_COMMITMENT, tx_expiration_block_num]
```

Comment:
Maybe not for this PR, but eventually we'll put delta commitment onto the stack as well, right?